sdk/communication/azure-communication-phonenumbers/azure/communication/phonenumbers/_phone_numbers_client.py

```diff
@@ -352,28 +347,27 @@ def list_available_localities(
     @distributed_trace
     def list_available_offerings(
         self,
-        country_code,  # type: str
-        **kwargs
-    ):
-        # type: (...) -> ItemPaged[PhoneNumberOffering]
+        country_code: str,
+        **kwargs: Any
```

Comment:
Do we want to use something like

*, phone_number_type

?

Reply:
Yeah, we should bump all these to proper keyword-only....
However this SDK is currently in beta, with a few new features on their way through preview - so I think I will ask the service team to make those changes as part of their next GA cycle.